README.rst

```diff
@@ -31,6 +31,10 @@
   :target: https://joss.theoj.org/papers/5b9024503f7bea324d5e738a12b0a108
   :alt: JOSS publication
 
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.3739558.svg
```

Comment:
is this zenodo batch different from the paper zenodo?

Reply:
They are separate:
- JOSS stores the paper and mints a DOI
- JOSS links to the zenodo archive that stores the code at the version when the JOSS paper was published

you can get to Zenodo from JOSS by clicking on "Software Archive" (see screenshot):

![image](https://user-images.githubusercontent.com/9084751/85889283-7799ff00-b7eb-11ea-8244-58282a0161d5.png)